charts/headlamp/tests/test_cases/oidc-create-secret.yaml

```diff
@@ -0,0 +1,9 @@
+config:
```

Comment:
It would be good if each of these test cases could explain what they are testing. Not sure the best way exactly...

Maybe with a comment in the test cases file? "It should ..." "This is testing that..."